scripts/export_inspector_findings.py

```diff
@@ -169,10 +171,17 @@ def column_alpha(self, col: int) -> str:
     def findings_sort(self, item: tuple[str, list[SummaryType]]) -> tuple[int, str]:
         score = 0
         weights = {'HIGH': 1, 'CRITICAL': 10}
-        for summary in item[1]:
+        vulnerability, summaries = item
+        for summary in summaries:
             count = len(summary['resources'])
             score += count * weights.get(summary['severity'], 0)
-        return score, item[0]
+        if vulnerability.startswith('CVE-'):
+            # Best secondary-sorting effort on CVE findings, vulnerability names
+            # not prefixed with 'CVE' may reflect an inaccurate secondary order.
+            id = vulnerability.split('-')[-1]
+            padded_id = '000000'[:abs(6 - len(id))] + id
```

Comment:
```suggestion
            padded_id = f'{id:0>6}'
```

With `abs`:
```
>>> for id in ['1234', '12345', '123456', '1234567', '12345678']:
...     print('000000'[:abs(6 - len(id))] + id)
    
001234
012345
123456
01234567
0012345678
```

With `max`:
```
>>> for id in ['1234', '12345', '123456', '1234567', '12345678']:
...     print('000000'[:max(0, 6 - len(id))] + id)
    
001234
012345
123456
1234567
12345678
```

With f-string:
```
>>> for id in ['1234', '12345', '123456', '1234567', '12345678']:
...     print(f'{id:0>6}')
    
001234
012345
123456
1234567
12345678
```

Also, (not needed if you use the f-string approach, but) remember a string can be multiplied by a number:
```
>>> '0' * 5
'00000'
```

Reply:
Good call on the f-string approach.
Thus far I've only seen 5 digits as part of the CVE ID which comes after the year, which is why I added six 0's (one extra for good measure).